### ISSUE TITLE
Confusion about your paper experimental session.

### ISSUE THREAD

**Issue by feifeibear** [alibaba/BladeDISC#85]
An interesting project. 
I glanced at your paper. The experimental section lists some performance comparison graphs. However, it does not indicate on which dimension of the input is dynamic.
in fact, the dynamic axis has been supported in TRT and onnxrt very well. Did you compare the results with them?
https://pytorch.org/tutorials/advanced/super_resolution_with_onnxruntime.html

**Comment by linearhit** [comment]
Hi， 
1,  More performance comparison will be provided later. It’s one of the major TODOs and we are working on it now. Both BladeDISC and other projects like TRT are rapidly evolving, some of our old digits can not be used for now. 
2, From our experience in using TensorRT (please correct if i’m wrong), its  supportability depends on the graph topology and shape range provided by the user, with awareness of the semantics of the dynamic dimensions, batchsize/seq_len for example. I would like to call this as “limited support”. So does some other vendor provided tools. Besides of DISC,  TVM Nimble is an example that “fully supports“ dynamic shape.
3, Although the current version of TRT can not be counted, it’s not a surprise for TRT to support this sooner or later, maybe in the near future. There is a strong demand from the users side to our knowledge. I believe this is one of the technology trends considered by most of the related projects unless restricted by the hardware.
4, For sure OnnxRT can fully support dynamic shape, it’s not a compiler, so does tensorflow/pytorch.

**Comment by feifeibear** [comment]
Thank you for your answer, I learned a lot. 
This project provides a lot of technical insight from a compiler perspective.
